e3sm_to_cmip/cmor_handlers/hur.py

```diff
@@ -19,13 +20,17 @@
 
 
 def write_data(varid, data, timeval, timebnds, index, **kwargs):
+
+    outdata = data[RAW_VARIABLES[0] ][index, :]
+    outdata[np.isnan(outdata)] = 1.e20
```

Comment:
I would define `1e20` as a static variable in `cmor_handlers/__init__.py` so that it is defined once and can be reused.
```
FILL_VALUE = 1e20
```

Instead of filling nans by subsetting using `np.isnan()`, you can use xarray's [`fillna()`](https://xarray.pydata.org/en/stable/generated/xarray.DataArray.fillna.html) method.
```
from e3sm_to_cmip.cmor_handlers import FILL_VALUE

outdata = outdata.fillna(FILL_VALUE)
```

Reply:
Also, are these the only handlers that require a `FILL_VALUE`?

Reply:
`outdata` is a numpy array not a xarray dataarray, so I use np.isnan() instead. Not sure if it is worthwhile to convert data type and use fillna()..

Reply:
Good idea to use static variable for fill value.

Reply:
Got it, I had a thought that it was possible a `np.ndarray` so `fillna()` might not work. 

I don't think it is worth the effort at this time, unless we refactor all the handlers.